docs/pages/deploy-a-cluster/helm-deployments/migration-kubernetes-1-25-psp.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Kubernetes 1.25 and PSP removal
+description: How to prepare the PodSecurityPolicy removal happening in Kubernetes 1.25
+---
+
+PodSecurityPolicies (PSPs) were deprecated in Kubernetes 1.22 and are removed
+in Kubernetes 1.25. This page explains the security implications of such changes
+for Teleport users, and what actions are required.
+
+The two Teleport charts `teleport-cluster` and `teleport-kube-agent` were relying
+on PodSecurityPolicies to provide an additional security level for Teleport pods.
+Their removal has two main consequences:
+
+- After upgrading to 1.25, Helm can end up with a corrupted state referencing
+  PSP objects. In this case, state has to be manually fixed.
```

Comment:
is `state` an ephemeral concept, or should this be something like:

```suggestion
  PSP objects. In this case, the cluster state has to be manually fixed.
```

Reply:
```suggestion
  PSP objects. In this case, the Helm release state has to be manually fixed.
```